src/containers/genePage/alleleTable.js

```diff
@@ -211,7 +216,7 @@ const AlleleTable = ({geneId}) => {
       style: {
         display: 'none',
       },
-      filterable: getDistinctFieldValue(resolvedData, 'filter.molecularConsequence'),
+      filterable: getDistinctFieldValue(resolvedData, 'filter.molecularConsequences'),
```

Comment:
@sibyl229 @jt15 I think this line needs to get reverted otherwise the rest of this PR looks good